acceptance-tests/src/test/java/uk/gov/di/test/step_definitions/Login.java

```diff
@@ -186,12 +187,42 @@ public void theExistingUserRequestsThePhoneOtpCodeTimes(int timesCodeIncorrect)
         }
     }
 
+    @When("the existing user enters the phone otp code {string} 6 times")
+    public void theExistingUserEntersThePhoneOtpCodeTimes(String incorrectCode) {
+
+        for (int i = 0; i < 5; i++) {
+            loginPage.enterSixDigitSecurityCode(incorrectCode);
+            findAndClickContinue();
+            waitForPageLoad("Error");
+            Assert.assertTrue(
+                    driver.findElement(By.className("govuk-error-summary")).isDisplayed());
```

Comment:
Should be using a ref to the PageObject for this, rather than the 'driver.findElement.....'